packages/react-components/react-color-picker-preview/stories/src/ColorPicker/ColorAndSwatchPicker.stories.tsx

```diff
@@ -0,0 +1,129 @@
+import * as React from 'react';
+import { tinycolor } from '@ctrl/tinycolor';
+import { makeStyles, Button, SwatchPicker, EmptySwatch, ColorSwatch, Label } from '@fluentui/react-components';
+import {
+  ColorPicker,
+  ColorSlider,
+  AlphaSlider,
+  ColorPickerProps,
+  ColorArea,
+} from '@fluentui/react-color-picker-preview';
+import type { SwatchPickerOnSelectEventHandler } from '@fluentui/react-components';
+
+const useStyles = makeStyles({
+  example: {
+    width: '300px',
+    display: 'flex',
+    flexDirection: 'column',
+    gap: '10px',
+  },
+  previewColor: {
+    width: '50px',
+    height: '50px',
+    borderRadius: '4px',
+    border: '1px solid #ccc',
+    margin: '10px 0',
+  },
+  button: {
+    marginRight: '8px',
+  },
+  input: {
+    display: 'block',
+    margin: '6px 0',
+  },
+  row: {
+    display: 'flex',
+    gap: '10px',
+    justifyContent: 'space-between',
+  },
+  sliders: {
+    display: 'flex',
+    flexDirection: 'column',
+    width: '80%',
+  },
+});
+
+const ITEMS_LIMIT = 8;
+const DEFAULT_COLOR_HSV = tinycolor('#2be700').toHsv();
+
+export const ColorAndSwatchPickerExample = () => {
+  const styles = useStyles();
+  const [color, setColor] = React.useState(DEFAULT_COLOR_HSV);
+  const [selectedValue, setSelectedValue] = React.useState('fff');
+  const [selectedColor, setSelectedColor] = React.useState('#fff');
```

Comment:
maybe it's worth to make these defaults as constants too :) 

```suggestion
  const [selectedValue, setSelectedValue] = React.useState(DEFAULT_SELECTED_VALUE);
  const [selectedColor, setSelectedColor] = React.useState(DEFAULT_SELECTED_COLOR);
```